GsaGH/Helpers/GsaAPI/EnumMappings/DiagramTypeMapping.cs

```diff
@@ -0,0 +1,11 @@
+﻿using GsaAPI;
+using Diagram = GsaGH.Parameters.Enums;
+
+namespace GsaGH.Helpers.GsaApi.EnumMappings {
+  internal class DiagramTypeMapping : EnumMapping<DiagramType, Diagram.DiagramType> {
+
+    public DiagramTypeMapping(
+      string description, DiagramType gsaApiEnum, Diagram.DiagramType gsaGhEnum) : base(description,
+      gsaApiEnum, gsaGhEnum) { }
+  }
```

Comment:
I am wondering if this `DiagramType gsaApiEnum` input may be what's causing the coverlet collector to fail? @tlmnrnhrdt @DominikaLos ?